Formula/libimagequant.rb

```diff
@@ -0,0 +1,34 @@
+class Libimagequant < Formula
+  desc "Palette quantization library extracted from pnquant2"
+  homepage "https://pngquant.org/lib/"
+  url "https://github.com/ImageOptim/libimagequant/archive/2.11.10.tar.gz"
+  sha256 "73fe142d8bb0fb637c4e9fe642ca4f7989c8cb5a034e8dd904de9bd10ba99688"
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "--enable-sse"
+    inreplace "Makefile", "$(SHAREDLIB).$(SOVER)", "libimagequant.$(SOVER).$(SOLIBSUFFIX)"
```

Comment:
needs to be fixed upstream

Reply:
I've proposed ImageOptim/libimagequant#22 to get this fixed upstream.

Is this really a blocker? It's just renaming `libimagequant.dylib.0` to `libimagequant.0.dylib`.

Reply:
I fixed a few more issues with the make system upstream (ImageOptim/libimagequant#24), I'll update this PR once the new version is tagged. It will also get rid of the manual static lib install and extra make steps.

Reply:
This review is stale and the requested changes have been integrated upstream, please update.